mpas_analysis/ocean/climatology_map_sose.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, config, mpasClimatologyTask,
                 '{}/SOSE_2005-2010_monthly_pot_temp_6000.0x' \
                 '6000.0km_10.0km_Antarctic_stereo.nc'.format(
                         observationsDirectory)
-            refFieldName = 'theta'
+            refFieldName = 'thetaSOSE'
```

Comment:
We also need to rename the variable in the observational data set to match this name.